ai-eval-service/main.py

```
@@ -87,14 +120,16 @@ def run_worker(payload: dict, timeout: int = 300):
                     f"Worker stderr (first 500 chars): {result.stderr[:500]}"
                 )
-                # Re-raise with context to ensure the parsing error isn't silently ignored
                 raise RuntimeError(
-                    f"Worker process failed and error response could not be parsed as JSON. "
-                    f"Worker stderr: {result.stderr[:500]}"
+                    "Worker process failed and error response could not be parsed as JSON"
                 ) from e
-            raise RuntimeError(f"Worker process failed: {result.stderr[:500]}")
+            logger.error(
+                f"Worker process failed with returncode {result.returncode}. "
+                f"Worker stderr (first 500 chars): {result.stderr[:500]}"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)